front/src/modules/timesStops/TimesStops.tsx

```diff
@@ -103,13 +104,12 @@ const TimesStops = ({
         } else {
           const newVias = updatedRows
             .filter((row, index) => !isEqual(row, rows[index]))
-            .map(
-              (row) =>
-                ({
-                  ...row,
-                  ...(row.arrival && { arrival: durationSinceStartTime(startTime, row.arrival) }),
-                }) as SuggestedOP
-            );
+            .map(({ onStopSignal, arrival, departure, ...row }) => ({
+              ...row,
+              arrival: durationSinceStartTime(startTime, arrival),
+              departure: durationSinceStartTime(startTime, departure),
```

Comment:
🔍 Out-of-scope change: adding `departure` management on the way to avoid "bad" cast as `SuggestedOP` it looks fine 🤞.